1-mandatory/2-quotegenerator/style.css

```diff
@@ -1 +1,50 @@
 /** Write your CSS in here **/
+
+.root {
+display: flex;
+justify-content: center;
+font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+}
+
+body{
+    background-color:rgb(236, 189, 35);
+}
+
+.container{
+display: flex;
+align-items: center;
+flex-direction: column;
+max-width: 80%;
+height: 80%;
+background-color: white;
+padding: 80px;
+margin: 150px;
+}
+
+h1{
+    display: flex;
+    text-align: center;
+    font-size: 10rem;
```

Comment:
10rem for a text size is pretty big one :) 
was that your real intention?

on my screen it's easier to read if i do these changes:
```css
.container {
    padding: 1rem;
    margin: 1rem;
}

h1 {
    font-size: 5rem;
}

p {
    font-size: 2rem;
}
```
note: only difference is shown above.

please don't forget about formatting.
make right mouse click on you css file and choose `Format Document`.